themes/gfsc/assets/sass/components/_signupsmall.sass

```diff
@@ -27,6 +27,8 @@
       align-items: center
       justify-content: space-between
       margin: 1.5rem 0 0.3rem 0
+  .signupsmall__label
+    display: none
```

Comment:
I think this won't work because it's the same as not having it in the DOM, screen readers should be able to read it and people should be able to click it to enter the field.

Maybe we should make the `p` in the box a label as that's describing what the input is for?

Reply:
I've also noticed this issue effects the form in the footer too

Reply:
hmmm. i did check it in the WAVE tool and it seemed happy. 

I don't think it looks bad having it visible.

![Screenshot_2023-07-10_13-57-00](https://github.com/geeksforsocialchange/gfsc-v3/assets/109223824/a6a130e7-e4bd-4d31-ac19-8b253e560cb0)

But that may be a content/layout question